### DISCUSSION TITLE
[Github Projects Beta] How can filter closed milestones?

### DISCUSSION THREAD

**Discussion by ramith** [community/community#29763] [Other Feature Feedback, Questions, & Ideas]
Hi all,
 I have a project (beta) created for a list of issues in a repository. In one of the views I group these issues by the milestone.
 Now the problem is there is no way I can hide/filter milestones that are closed.
Any ideas?

**Comment by filipjonckers** [comment]
Have the same issue.
The Table view shall be able to filter out closed milestones, a filter such as milestone-status: [open/closed/...) should be available.

**Comment by ramith** [comment]
+1 for @filipjonckers suggestion.

**Comment by EddisonShyi** [comment]
+1, need filter to hide closed milestones
